css/.csscomb.json

```diff
@@ -13,11 +13,9 @@
     "combinator-space": [" ", " "],
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": null,
```

Comment:
Perhaps we should disable [this rule](https://github.com/mobify/mobify-code-style/blob/develop/css/.sass-lint.yml#L56)